### PR TITLE
Ruby3 Support

### PR DESCRIPTION
**Pull request by cjcolvar**
This PR adds support for ruby 3 by upgrading ldp and ActiveTriples as well switching to bixby in order to upgrade rubocop to a version with support for ruby 3.  The upgrade to bixby required extensive styling fixes, but brings ActiveFedora in line with the rest of the core components.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -4,6 +4,7 @@ source "https://rubygems.org"
 
 gemspec path: File.expand_path('..', __FILE__)
 
+gem 'active-triples', github: 'samvera-labs/activetriples', branch: 'merge-gitlab-upstream'
```
Comment (jrgriffiniii) [inline Gemfile:7]: 👍

File: active-fedora.gemspec [hunk 2]
```diff
@@ -8,11 +9,11 @@ Gem::Specification.new do |s|
...
   s.email       = ["samvera-tech@googlegroups.com"]
-  s.homepage    = %q{https://github.com/samvera/active_fedora}
-  s.summary     = %q{A convenience libary for manipulating documents in the Fedora Repository.}
-  s.description = %q{ActiveFedora provides for creating and managing objects in the Fedora Repository Architecture.}
+  s.homepage    = 'https://github.com/samvera/active_fedora'
+  s.summary     = 'A convenience libary for manipulating documents in the Fedora Repository.'
+  s.description = 'ActiveFedora provides for creating and managing objects in the Fedora Repository Architecture.'
   s.license = "Apache-2.0"
-  s.required_ruby_version = '>= 2.4'
+  s.required_ruby_version = '>= 2.6'
```
Comment (jrgriffiniii) [inline active-fedora.gemspec:16]: 👍

File: spec/unit/finder_methods_spec.rb [hunk 3]
```diff
@@ -18,7 +18,7 @@ def self.default_sort_params
   end
 
   let(:finder_class) do
-    this = self
+    # this = self
```
Comment (jrgriffiniii) [inline spec/unit/finder_methods_spec.rb:21]: Not a blocker, but this should probably be deleted in a future PR.

**Review by jrgriffiniii — APPROVED** [review]
Thank you very much for this incredible amount of labor @cjcolvar !!! It is very greatly appreciated!

File: active-fedora.gemspec (hunk 2 above)
Comment (dlpierce) [inline active-fedora.gemspec:16]: Should this match the minimum tested ruby version?

Comment (cjcolvar) [reply active-fedora.gemspec:16]: I'm not sure it really needs to, but I could add testing back in for ruby 2.6.

**Review by dlpierce — APPROVED** [review]

**Pull request merged**